public/login.html

```
@@ -8,140 +9,177 @@
-          <input id="r-password" type="password" autocomplete="new-password" required minlength="6"
-            class="w-full px-4 py-2.5 rounded-xl border border-slate-200 focus:outline-none focus:ring-2 focus:ring-brand/40 text-sm" placeholder="min 6 characters" />
-        </div>
-        <p id="register-err" class="text-red-500 text-sm hidden"></p>
-        <button type="submit" class="w-full bg-brand text-white font-semibold py-2.5 rounded-xl hover:bg-brand-dark transition text-sm">Create Account</button>
-      </form>
+      <div id="panel-login" class="p-8">
+        <h2 class="text-xl font-bold text-slate-800 mb-6">Welcome back!</h2>
+        <form id="form-login" class="space-y-4" novalidate>
+          <div>
+            <label class="block text-sm font-medium text-slate-700 mb-1">Username</label>
+            <input id="l-username" type="text" autocomplete="username" required
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)